metomi/rose/suite_engine_procs/cylc.py

```diff
@@ -65,35 +72,49 @@ def get_suite_dir_rel(cls, suite_name, *paths):
         """
         return os.path.join(cls.SUITE_DIR_REL_ROOT, suite_name, *paths)
 
-    def get_suite_jobs_auths(self, suite_name, cycle_name_tuples=None):
-        """Return remote ["[user@]host", ...] for submitted jobs."""
-        # TODO: reimplement for Cylc8?
-        # https://github.com/metomi/rose/issues/2445
-        self.handle_event(
-            Exception(
-                'WARNING: Rose cannot currently inspect the platform a Cylc'
-                ' task has or will run on.\n'
-                ' https://github.com/metomi/rose/issues/2445'
-            )
-        )
-        return []
-
-    def get_task_auth(self, suite_name, task_name):
-        """Return [user@]host for a remote task in a suite.
-
-        Or None if task does not run remotely.
+    def get_suite_jobs_auths(
+        self, suite_name: str, cycle_name_tuples: Tuple[Any] = None
+    ) -> List[str]:
```

Comment:
Note that mypy is not enabled for this repo, so this is not checked by GH Actions
```suggestion
    def get_suite_jobs_auths(
        self, suite_name: str, cycle_name_tuples: Optional[Iterable[Tuple[str, str]]] = None
    ) -> List[str]:
```

Reply:
This suggestion was implemented before, but has now gone missing